src/main/java/tests/DataConverterTest.java

```diff
@@ -0,0 +1,61 @@
+package tests;
+
+import core.basesyntax.DataConverter;
+import core.basesyntax.DataConverterImpl;
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterTest {
+
+    private final List<String> rawData = Arrays.asList(
+            "type,fruit,quantity",
+            "b,banana,20",
+            "s,banana,100",
+            "p,banana,13",
+            "r,banana,10"
+    );
+    private final DataConverter dataConverter = new DataConverterImpl();
+    private final List<FruitTransaction> transactions = dataConverter.convertToTransaction(rawData);
+
+    @Test
+    public void data_converter_size_test() {
+        Assertions.assertEquals(4, transactions.size());
```

Comment:
Consider using a more descriptive name for your test methods. The method name should express what the test is doing and what result is expected. For example, 'data_converter_size_test' could be renamed to 'convertToTransaction_ShouldReturnCorrectSize_WhenValidDataProvided'.